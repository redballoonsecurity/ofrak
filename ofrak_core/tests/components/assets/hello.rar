version https://git-lfs.github.com/spec/v1
oid sha256:3f05886d6a27bc9c93cd0b8ad7a3583997e69647a310e28bdb19b29fbe276199
size 78
