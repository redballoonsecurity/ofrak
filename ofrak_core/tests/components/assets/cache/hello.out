version https://git-lfs.github.com/spec/v1
oid sha256:f154a4d1ed1a55ee192be2dd343fd49a97572d9500586049c3bf8f53a45414ea
size 44816
